product-mini/platforms/nuttx/wamr.mk

```diff
@@ -274,7 +274,11 @@ endif
 
 ifeq ($(CONFIG_INTERPRETERS_WAMR_GLOBAL_HEAP_POOL),y)
 CFLAGS += -DWASM_ENABLE_GLOBAL_HEAP_POOL=1
+ifdef WAMR_BUILD_GLOBAL_HEAP_SIZE
```

Comment:
It doesn't make sense since all variables in NuttX are from Kconfig and with a 'CONFIG_' prefix,
and in NuttX, we are already make the globl heap size configurable.

So we can revert the change in wamr.mk.